app/src/main/java/com/zendrive/zendrivesdkdemo/MainFragment.java

```diff
@@ -77,29 +91,34 @@ public void onClick(View v) {
                 Zendrive.teardown();
 
                 // Initialize zendrive sdk
-                String driverId = "someDriverId";
-                MainFragment.this.initializeZendriveSDK(driverId);
+                MainFragment.this.initializeZendriveSDK();
             }
         });
 
         return rootView;
     }
 
-    public void initializeZendriveSDK(final String driverId) {
+    public void initializeZendriveSDK() {
 
         // Get user attributes (Optional)
         ZendriveDriverAttributes userAttributes = new ZendriveDriverAttributes();
-        userAttributes.setFirstName("FirstName");
-        userAttributes.setLastName("LastName");
-        userAttributes.setEmail("userEmail");
-        final String applicationKey = applicationKeyEditText.getText().toString();
+        if (!driverFirstName.equals("")) {
```

Comment:
("").equals(driverFirstName) -- this would also look for null values just in case.